### DISCUSSION TITLE
knapsack problem

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#63] [Esercizi A-L]
Dato uno zain do capacità $C$ e dati gli oggetti $x_1, x_2, ..., x_n$ con peso $p(x_i) = p_i$ e valore $v(x_i) = v_i$ scrivere lo pseudocodice di un algoritmo che scegli un sottoinsieme di oggetti che massimizza il valore totale, e rispetta la capacità in $O(n \cdot C)$

**Comment by Traster-uni** [comment]
![WhatsApp Image 2024-06-13 at 6 58 59 PM](https://github.com/sapienzastudentsnetwork/progettazione-di-algoritmi/assets/84441861/af01c2ae-97b5-4335-8f0a-eb86666678ef)
